recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "scalene" %}
+{% set version = "1.5.9.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scalene-{{ version }}.tar.gz
+  sha256: a3adfe397e1e379091e3405c692a178291ebe8c2d3721dbc02bfec5ecc347ef7
+
+build:
+  number: 0
+  skip: true   # [py<37]
```

Comment:
```suggestion
  skip: true   # [py<38]
```

Reply:
https://github.com/plasma-umass/scalene/blob/master/setup.py#L169?

Reply:
[Support for Python 3.7 has been removed.](https://github.com/plasma-umass/scalene/commit/9b7cb2dfe4bd3f2da6f372c1bf5b3bd95ea4b333)

Reply:
Oops

Reply:
My brain had a bug evaluating `<` and `>=`